addons/semicolon/inverted.css

```diff
@@ -0,0 +1,5 @@
+.semicolon {
+  margin: 0;
+  background: black;
+  color: white;
+}
```

Comment:
Doing this is not a good approach... The better way to do this would be in the dark mode css where you should try testing setting the background of the body black

Reply:
ah